Description: lintian shows "python-script-but-no-python-dep" when "python2" is
used in the hasbang. It can be changed to "python", as long Python 2 is the
default interpreter (instead of Python 3).
Author: Tiago Ilieve <tiago.ilieve@gmail.com>

--- a/pythonpy/__main__.py
+++ b/pythonpy/__main__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
 from __future__ import (unicode_literals, absolute_import,
                         print_function, division)
 import sys
--- a/pythonpy/pycompleter.py
+++ b/pythonpy/pycompleter.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
 from __future__ import (unicode_literals, absolute_import,
                         print_function, division)
 import sys
